packages/components/note/src/Note.tsx

```diff
@@ -73,11 +77,19 @@ export const Note = React.forwardRef<HTMLElement, ExpandProps<NoteProps>>(
       onClose,
       testId = 'cf-ui-note',
       title,
+      startIcon,
       ...otherProps
     } = props;
 
     const styles = getNoteStyles();
 
+    const iconContent = (icon: React.ReactElement) =>
+      React.cloneElement(icon, {
+        size: icon.props.size ?? `${title ? 'medium' : 'small'}`,
```

Comment:
Do you see a need to open up for size customisation?

Reply:
Not required for our first use case of using a custom component, let's keep the defaults for now.